hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/CommandQueue.java

```diff
@@ -38,9 +40,8 @@
  * there where queued.
  */
 public class CommandQueue {
-  // This list is used as default return value.
-  private static final List<SCMCommand> DEFAULT_LIST = new ArrayList<>();
   private final Map<UUID, Commands> commandMap;
+  private final Map<UUID, Map<SCMCommandProto.Type, Integer>> summaryMap;
```

Comment:
Would it be better if we add another class to handle Commands for one Datanode separately? Like something like DatanodeQueue. So summary and commands can go together in one place.

Reply:
I guess that is something we could do in the future if there were more things to track on a datanode by datanode basis. I'd like to avoid too much change here, and a class per DN like a DatanodeQueue would look fairly similar I think. Something like this could easily be added later if we need it, as it will be behind the CommandQueue interface, and the changes would just be internal to the class.

Reply:
Maybe the summary could go into the commands class, since we already have that

Reply:
I see we are doing the same map operation twice here

Reply:
Yea putting the summary into the commands class might make it tidier. I will try that tomorrow.